packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -28,24 +26,23 @@ int topMatchesLimit = defaultTopMatchesLimit;
 
 const double _searchControlDividerHeight = 24.0;
 
-mixin SearchControllerMixin<T extends DataSearchStateMixin> {
+mixin SearchControllerMixin<T extends DataSearchStateMixin?> {
   final _searchNotifier = ValueNotifier<String>('');
   final _searchInProgress = ValueNotifier<bool>(false);
 
   /// Notify that the search has changed.
   ValueListenable get searchNotifier => _searchNotifier;
```

Comment:
add type `<String>` to this one too for consistency

Reply:
done